.Rbuildignore

```diff
@@ -6,3 +6,4 @@
 ^issue[0-9]+$
 ^examples$
 ^data-raw$
+^vignettes$
```

Comment:
I'm not sure that we want to have the entire `vignettes` folder included? If you look at `ggplot2`, they have [some directories](https://github.com/tidyverse/ggplot2/blob/375a24b58c156c2bbb5bf18a027461e2a03d4560/.Rbuildignore#L25-L26) listed in the `.Rbuildignore`, but not [all](https://github.com/tidyverse/ggplot2/tree/375a24b58c156c2bbb5bf18a027461e2a03d4560/vignettes) of them 

```suggestion

```

Reply:
Not sure what's the best here. In any case, I implemented your suggestion in https://github.com/JMSLab/eventstudyr/pull/30/commits/e43ce94b1cb14f8d2ce45cf2187864cf6cb28475 and Actions ran correctly after the commit. Workflow run is [here](https://github.com/JMSLab/eventstudyr/actions/runs/4515115218)